samples/unit-tests/axis/overscroll/demo.js

```diff
@@ -74,7 +75,10 @@
             options = getOptions();
```

Comment:
Can this line `options = getOptions();`  be removed safely?

Reply:
Wow, good catch!